lib/ht/search_client/test.rb

```diff
@@ -89,6 +102,14 @@ def stub_request(endpoint, params)
 
       WebMock.stub_request(:get, service_uri.to_s)
     end
+
+    def stub_post_request(endpoint, params)
```

Comment:
This is not very DRY regarding to `stub_request` no ? why this one also needs the `with` ? I thought this was handled outside of this method ?